test/CrossImport/skip-submodules.swift

```diff
@@ -4,6 +4,11 @@
 // RUN: %FileCheck %s < %t.txt
 // RUN: %FileCheck --check-prefix=NEGATIVE %s < %t.txt
 
+// FIXME: This test does not pass because another optimization is now skipping
+// this work entirely. It needs to be redesigned to use modules which have both
+// submodules *and* cross-import overlays.
+// XFAIL: *
```

Comment:
If someone does merge this, please file a radar on me and add the radar number here.

Reply:
Deleted this test in favor of a more useful one.